tests/client/core/api/test_api.js

```diff
@@ -164,6 +164,25 @@ describe('api', () => {
           mockWindow.verify();
         });
     });
+
+    it('sends the config when set', () => {
+      sinon.stub(config, 'get').withArgs('fxaConfig').returns('my-config');
+      mockWindow
+        .expects('fetch')
+        .withArgs('https://addons.mozilla.org/api/v3/accounts/login/?config=my-config&lang=fr', {
+          body: '{"code":"my-code","state":"my-state"}',
+          credentials: 'include',
+          headers: { 'Content-type': 'application/json' },
+          method: 'post',
+        })
+        .once()
+        .returns(mockResponse());
+      return api.login({ api: { lang: 'fr' }, code: 'my-code', state: 'my-state' })
+        .then((apiResponse) => {
+          assert.strictEqual(apiResponse, response);
+          mockWindow.verify();
```

Comment:
Most of this testing is redundant because it's already covered in https://github.com/mstriemer/addons-frontend/blob/908ec1737abb89ab11f08be7700536bb9ab3a392/tests/client/core/api/test_api.js#L150 The suite will be harder to maintain (as logic changes) when the same logic is tested in multiple places.

All you need to do is check the query string param (or full URL) because that's the only thing the config will influence.